### PR TITLE
Implement front end components and styling until contact host feature

### PR DESCRIPTION
**Pull request by sowmyaneni**

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -1,24 +1,70 @@
...
             }
         });
     }
+    displaySpaceInfo() {
+        this.setState({
+           displaySpaceText: !this.state.displaySpaceText
+        })
+    }
     render() {
+        let bedroom = 'bedrooms';
```
Comment (willchan8) [inline client/src/components/App.jsx]: Maybe use ternary operators to combine Line 33 with 44-46, Line 34 with 47-49, and Line 35 with 50-52.

File: client/src/components/App.jsx [hunk 2]
```diff
@@ -1,24 +1,70 @@
...
+import SparklingClean from './SparklingClean.jsx';
+import GreatLocation from './GreatLocation.jsx';
+import SelfCheckin from './SelfCheckin.jsx';
+
 class App extends React.Component {
     constructor() {
         super();
         this.state = {
-            listings: []
+            listings: [],
```
Comment (willchan8) [inline client/src/components/App.jsx]: Is listings an array or an object?

File: client/src/components/App.jsx [hunk 3]
```diff
@@ -1,24 +1,70 @@
...
+        if (this.state.listings.numberOfBedrooms === 1) {
+            bedroom = 'bedroom';
+        }
+        if (this.state.listings.numberOfBaths === 1) {
+            bath = 'bath';
+        }       
+        if (this.state.listings.numberOfBeds === 1) {
+            bed = 'bed';
+        }
+        if (this.state.listings.isSuperhost === true) {
```
Comment (willchan8) [inline client/src/components/App.jsx]: Maybe use destructuring to prevent repeat of `this.state`.

Comment (lenacha) [reply client/src/components/App.jsx]: Following airbnb style guide makes the code more clear and simple!

Comment (sowmyaneni) [reply client/src/components/App.jsx]: Refactored code to include destructuring. i will upload the next version once I finish the modal display for amenities.

File: client/src/components/App.jsx (hunk 1 above)
Comment (sowmyaneni) [reply client/src/components/App.jsx]: Refactored to use ternary operators.

File: client/src/components/App.jsx (hunk 2 above)
Comment (sowmyaneni) [reply client/src/components/App.jsx]: It's an array of objects that includes all 100 listings.

**Pull request merged**